README.md

```diff
@@ -28,92 +20,10 @@ git clone git@github.com:benkrikler/LLP_generation.git
 cd LLP_generation
 ```
 
-Checkout CMSSW version `8_0_21`:
-```bash
-export SCRAM_ARCH=slc6_amd64_gcc530
-cmsrel CMSSW_8_0_21
-cd CMSSW_8_0_21/src
-cmsenv
-mkdir -p Configuration/GenProduction/python/
-cp ../../fragments/T1qqqqLL_fragment.py Configuration/GenProduction/python/
-scram b -j 9
-cd -
-```
-
 ## Commands to produce the CMSSW config files
 This repository already contains the files produced by the following commands.
 But for reference, here are the commands for each step.
```

Comment:
Should probably change that last line and mention the two new READMEs for full / fast sim?